### DISCUSSION TITLE
archived:false does not work on code search

### DISCUSSION THREAD

**Discussion by piyushmor** [community/community#32740] [Other Feature Feedback, Questions, & Ideas] [answered]
we have seen that the filter for archived false is not working on the code search across repositories. is this something that can be easily achieved? would be really helpful.

**Comment by colinwm** [comment, marked as answer]
There is a way to search for non-archived repositories: use `NOT is:archived`

**Comment by ls-ivan-kiselev** [comment]
Doesn't really work for me O_o

**Reply by colinwm** [reply]
Are you using cs.github.com? To clarify, the `NOT is:archived` query only works there. For the old github.com code search, you need to use `archived:false`
